dotnet-authserver/src/TeacherIdentity.AuthServer/Migrations/TeacherIdentityServerDbContextModelSnapshot.cs

```diff
@@ -798,7 +802,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<int>("WebHookMessageTypes")
                         .ValueGeneratedOnAdd()
                         .HasColumnType("integer")
-                        .HasDefaultValue(3)
+                        .HasDefaultValue(7)
```

Comment:
it's because there is a new value in the flag so "All" becomes bigger .. I didn't think it did any harm as it won't affect existing webhooks and new ones shouled default to "All" really .. even though that will actually get set from whatever is entered in the UI in reality